[docs]Add marked source in tutorial doc

shogunsea opened the pull request:
In `Getting started` section, add source for `marked.min.js` so that following the steps from tutorial won't lead to this error:

![image](https://cloud.githubusercontent.com/assets/2238944/11449273/bd15d998-9534-11e5-9b9e-278678969630.png)

shogunsea commented:
just found out in https://github.com/facebook/react/pull/3663 `marked` is included in `Adding Markdown` section, however I still feels that it's confusing to not have it included and telling people: 
`When you load this for the first time without any changes, you'll see the finished product of what we're going to build.`

zpao commented:
I'm ok adding this here but only if we fix the rest of the wording where we talk about adding this later. Could you do that?

shogunsea commented:
@zpao sure, I'll do that

zpao commented on docs/docs/tutorial.md:
```diff
@@ -221,7 +222,7 @@ Note that we have passed some data from the parent `CommentList` component to th
 Markdown is a simple way to format your text inline. For example, surrounding text with asterisks will make it emphasized.
 
-First, add the third-party library **marked** to your application. This is a JavaScript library which takes Markdown text and converts it to raw HTML. This requires a script tag in your head (which we have already included in the React playground):
+In this tutorial we use a third-party library **marked** which takes Markdown text and converts it to raw HTML. This requires a script tag in your head (which we have already included in the React playground):
```
A couple changes. First, let's change the 2nd sentence here.

> This requires a script tag in your head (which we have already included in the React playground):

Let's just make that:

> We already included this library with the original markup for the page, so we can just start using it.

And then let's get rid of this whole HTML code block below. It's just noise since the script is already there.

zpao commented on docs/docs/tutorial.md:
````diff
@@ -236,7 +237,7 @@ First, add the third-party library **marked** to your application. This is a Jav
 ```
 
-Next, let's convert the comment text to Markdown and output it:
+Let's convert the comment text to Markdown and output it:
````
Then we can just put this sentence into the preceding paragraph since they are both super short and flow pretty well.

zpao commented:
There's one more in the paragraph above your first change.

> It should look something like this (with perhaps some minor differences, we'll add an additional `<script>` tag later):

We don't need to say that we're adding an additional script tag later.

And then lastly, could you squash these into a single commit for condensed history.

shogunsea commented:
@zpao great thoughts on this! I've addressed your comments,  assuming the entire sentence in bracket here is unnecessary:
>
It should look something like this (with perhaps some minor differences, we'll add an additional <script> tag later):
>

Let me know if this needs more work, thanks!

zpao commented:
Looks great. Thanks!
